policy/modules/system/logging.if

```diff
@@ -681,10 +681,9 @@ interface(`logging_send_syslog_msg',`
 		# Allow systemd-journald to check whether the process died
 		allow syslogd_t $1:process signull;
 
-		ifdef(`distro_redhat',`
-			kernel_dgram_send($1)
-		')
 	')
+
+	kernel_dgram_send($1)
```

Comment:
This looks like it is at the wrong level block. It should still be in `init_systemd`, right?